routes.go

```diff
@@ -224,13 +227,18 @@ func handlePrimaryEndpoint(in primaryEndpointIn) {
 }
 
 func handleWebhookRoutes(in handleWebhookRoutesIn) error {
-	if in.AddWebhookHandler != nil && in.GetAllWebhooksHandler != nil {
-		fixV2Middleware, err := fixV2Duration(sallust.Get, in.WebhookConfig.Validation.TTL, in.V2AddWebhookHandler)
-		if err != nil {
-			fmt.Fprintf(os.Stderr, "Failed to initialize v2 endpoint middleware: %v\n", err)
-			return err
+	if in.PreviousVersion {
```

Comment:
are we getting rid of fixV2Middleware once new webhook and kafka updates are completed?
this right now is checking if it's v2 and if yes we use the fixv2middleware otherwise it's not used